AutoUpdate/CKAN-autoupdate.csproj

```diff
@@ -32,6 +32,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="log4net, Version=1.2.13.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
```

Comment:
This seems to be the error in mono, It can parse this line (repeated for each project file). But this was generated by Visual Studio so i am hesitant to edit it back without knowing what all of it does? 

Anyone with more expertise than me tell me if its fine to change back to:
`<Reference Include="log4net">`